### ISSUE TITLE
Use of uninitialized value in hash element at Spreadsheet/ParseXLSX.pm line 477.

### ISSUE THREAD

**Issue by vfilatov** [doy/spreadsheet-parsexlsx#41]
```
$ file test.xlsx 
test.xlsx: Microsoft OOXML
$ ./ParseXLSX.pl test.xlsx > /dev/null
Use of uninitialized value in hash element at Spreadsheet/ParseXLSX.pm line 477.

Workaround
Spreadsheet/ParseXLSX.pm:
479c479
<             $fill{$_->att('patternType')},
---
>             $fill{$_->att('patternType')//'none'},
```

**Comment by doy** [comment]
Can you provide me with `test.xlsx`?

**Comment by vfilatov** [comment]
Sorry, for delay...
Here it is.

Thanks,
Val

**Comment by vfilatov** [comment]
I checked v0.18 bug is still there

**Comment by doy** [comment]
Sorry - Github issue emails don't transmit attachments. Can you upload the file somewhere instead?

**Comment by doy** [comment]
Fixed in 5339faa.
